CMP1903_A1_2324/Die.cs

```diff
@@ -14,9 +14,17 @@ internal class Die
          */
 
         //Property
-
+        int currentRoll = 0;
```

Comment:
Should specify private even though it is implicit.
Private variables also must be in _camelCase, "_currentRoll".